alembic/versions/575f99caf2d4_solve_issues_active_flag_to_sites_166_.py

```diff
@@ -0,0 +1,230 @@
+"""solve issues active flag to sites-166,add filter on api read-173
+
+Revision ID: 575f99caf2d4
+Revises: ea0672033541
+Create Date: 2024-11-25 20:43:55.866383
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '575f99caf2d4'
+down_revision = 'ea0672033541'
```

Comment:
oh, and becasue of https://github.com/openclimatefix/pv-site-datamodel/pull/180, you might need to update this - sorry

Reply:
would you please tell me what i need to do further

Reply:
I think if you 
1. change down_version to d5c9e13a8b66
2. merge `main` branch back in